ui/app/AppLayouts/Wallet/stores/CollectiblesStore.qml

```diff
@@ -6,6 +6,7 @@ import StatusQ.Models 0.1
 import utils 1.0
 
 import SortFilterProxyModel 0.2
+import StatusQ.Core.Utils 0.1
```

Comment:
This should be imported under an alias `... as SQUtils` and the code adjusted, otherwise this might lead into some nasty side effects